vizro-core/src/vizro/actions/_actions_utils.py

```diff
@@ -120,11 +120,7 @@ def _get_parametrized_config(
     for target in targets:
         # TODO - avoid calling _captured_callable. Once we have done this we can remove _arguments from
         #  CapturedCallable entirely.
-        graph_config = (
-            deepcopy(model_manager[target].figure._arguments)  # type: ignore[attr-defined]
-            if hasattr(model_manager[target], "figure")
-            else deepcopy(model_manager[target].table._arguments)  # type: ignore[attr-defined]
-        )
+        graph_config = deepcopy(model_manager[target]._callable_component._arguments)  # type: ignore[attr-defined]
```

Comment:
Sorry I wasn't clear here but what I meant was the `_callable_component: str = PrivateAttr("figure")` and then here it would be `getattr`. The `_callable_component` is just telling us what the relevant field is, not a replacement or copy of that field.